tests/ops/util/test_tc_string.py

```diff
@@ -6,174 +6,28 @@
 from fides.api.models.privacy_notice import UserConsentPreference
 from fides.api.models.sql_models import PrivacyDeclaration, System
+from fides.api.schemas.privacy_preference import TCStringFidesPreferences
 from fides.api.util.tcf.experience_meta import (
     TCFVersionHash,
     _build_tcf_version_hash_model,
     build_tcf_version_hash,
 )
-from fides.api.util.tcf.tc_mobile_data import build_tc_data_for_mobile
+from fides.api.util.tcf.tc_mobile_data import (
+    build_tc_data_for_mobile,
+    convert_tc_string_to_mobile_data,
+)
 from fides.api.util.tcf.tc_model import CMP_ID, convert_tcf_contents_to_tc_model
-from fides.api.util.tcf.tc_string import TCModel, build_tc_string
+from fides.api.util.tcf.tc_string import (
+    TCModel,
+    build_tc_string,
+    decode_tc_string_to_preferences,
+)
 from fides.api.util.tcf.tcf_experience_contents import get_tcf_contents
 
 
-@pytest.fixture(scope="function")
-def captify_technologies_system(db: Session) -> System:
-    """Add system that only has purposes with Consent legal basis"""
-    system = System.create(
-        db=db,
-        data={
-            "fides_key": f"captify_{uuid.uuid4()}",
-            "vendor_id": "2",
-            "name": f"Captify",
-            "description": "Captify is a search intelligence platform that helps brands and advertisers leverage search insights to improve their ad targeting and relevance.",
-            "organization_fides_key": "default_organization",
-            "system_type": "Service",
-            "uses_profiling": False,
-            "legal_basis_for_transfers": ["SCCs"],
-        },
-    )
-
-    for data_use in [
-        "functional.storage",  # Purpose 1
-        "marketing.advertising.negative_targeting",  # Purpose 2
-        "marketing.advertising.frequency_capping",  # Purpose 2
-        "marketing.advertising.first_party.contextual",  # Purpose 2
-        "marketing.advertising.profiling",  # Purpose 3
-        "marketing.advertising.first_party.targeted",  # Purpose 4
-        "marketing.advertising.third_party.targeted",  # Purpose 4
-        "analytics.reporting.ad_performance",  # Purpose 7
-        "analytics.reporting.campaign_insights",  # Purpose 9
-        "functional.service.improve",  # Purpose 10
-        "essential.fraud_detection",  # Special Purpose 1
-        "essential.service.security"  # Special Purpose 1
-        "marketing.advertising.serving",  # Special Purpose 2
-    ]:
-        # Includes Feature 2, Special Feature 2
-        PrivacyDeclaration.create(
-            db=db,
-            data={
-                "system_id": system.id,
-                "data_use": data_use,
-                "legal_basis_for_processing": "Consent",
-                "features": [
-                    "Link different devices",
-                    "Actively scan device characteristics for identification",
-                ],
-            },
-        )
-
-    db.refresh(system)
-    return system
```

Comment:
Just moved these to application fixtures to be shared more broadly. These are real vendors with the important TCF pieces defined (data uses, features) that are used for testing the TC string is constructed properly -